src/main/java/com/atlassian/db/replica/api/aurora/AuroraReplicasDiscoverer.java

```diff
@@ -0,0 +1,55 @@
+package com.atlassian.db.replica.api.aurora;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.atlassian.db.replica.api.aurora.AuroraEndpoints.replicaUrl;
+
+/**
+ * It allows to pull Aurora Replicas cluster information
+ */
+public class AuroraReplicasDiscoverer {
+    private final Connection connection;
+
+    public AuroraReplicasDiscoverer(Connection connection) {
+        this.connection = connection;
+    }
+
+    /**
+     * It provides list of replica urls based on reader endpoint and server ids
+     * @param readerEndpoint - Aurora reader endpoint
+     * @return list of replicas jdbcUrls
+     * @throws SQLException
+     */
+    public List<String> fetchReplicasUrls(String readerEndpoint) throws SQLException {
+        return fetchReplicaServerIds()
+            .stream()
+            .map(serverId -> replicaUrl(
+                readerEndpoint,
+                serverId
+            )).collect(Collectors.toList());
```

Comment:
I cannot resist fixing ad hoc chopping 😝 
```suggestion
            .map(serverId -> replicaUrl(readerEndpoint, serverId))
            .collect(toList());
```